templates/main/group_builds.html.ep

```diff
@@ -22,6 +22,9 @@
                         <i class="tag fa fa-tag" title="<%= $tag->{type}; %>"><%= $tag->{description} %></i>
                     </span>
                 % }
+                % if ($children) {
+                 <a href="<%= url_for_test_overview($build_res) %>" title="View Test Results"><i class="fas fa-link"></i></a>
```

Comment:
This is the link on each subgroup build result, right? I think this is not needed, only the link on the parent group in your current implementation